src/_redux/main/nodeTree/event/slice.ts

```diff
@@ -15,6 +15,7 @@ import { TNodeEventReducerState } from "./types";
 const nodeEventReducerInitialState: TNodeEventReducerState = {
   currentFileContent: "",
   selectedNodeUids: [],
+  currentFileUid: "",
```

Comment:
Why we have moved it from file to node?

Reply:
@atulbhatt-system32 
This is added to have a specific file in the history.

Previously, after double-clicking on a web component and performing a certain action, the web component code was applied to the design file, but by adding it to the history, we can filter this out. 
This is also necessary for a seamless switching between files.

Reply:
But why are we keeping it in node history instead of file